dev/io.openliberty.org.jboss.resteasy.common.ee11/src/org/jboss/resteasy/plugins/providers/sse/client/SseEventSourceImpl.java

```diff
@@ -378,10 +378,26 @@ public void run() {
                     break;
                 }
                 try {
-                    // Process the event
-                    onEvent(event);
-                    // Read next event
+	            // Liberty change start
```

Comment:
Normally I'd agree, but in this case I'm essentially setting it back to what it was in ee10.   I thought I could go with what was in RESTEasy for EE11 but there are still "differences" in how we read the spec and I didn't want to cause a regression since some of our FAT failed.